modules/timeline.xql

```diff
@@ -15,25 +15,35 @@ response:set-header("Cache-Control", "no-cache"),
 
 let $start := request:get-parameter("start",1)
 let $count := request:get-parameter("count",10)
-let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist;lts&amp;start=" || $start || "&amp;count=" || $count
-let $data := httpclient:get(xs:anyURI($timelineURLString), false(), ())
-let $entries :=
-    for $entry in $data/httpclient:body//atom:entry
+let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist&amp;start=" || $start || "&amp;count=" || $count
+let $data := http:send-request(
+    <http:request method='get' />,$timelineURLString)
+  let $entries :=
+    for $entry in $data//atom:entry
     let $date := ($entry/atom:updated, $entry/atom:published)[1]
     order by xs:dateTime($date) descending
     return $entry
 
+return $entries
```

Comment:
This return looks misplaced, I would expect that nothing below this line will be executed.

Reply:
I know, in fact in order to test it I had to take it out. Nevertheless it's in the original file, so I thought it was just me not knowing enough.